### PR TITLE
#646 remove init

### PR DESCRIPTION
**Pull request by nimshi89**
Removing the imports from the init files

### CODE REVIEW

File: src/csvcubed/readers/__init__.py [hunk 1]
```diff
@@ -1,6 +0,0 @@
-"""
```
Comment (robons) [inline src/csvcubed/readers/__init__.py:1]: Can you stick this back?

File: src/csvcubed/models/rdf/__init__.py [hunk 2]
```diff
@@ -1,3 +0,0 @@
-"""
-Contains Models for mapping data to RDF which are unique to the `csvcubed` package.
```
Comment (robons) [inline src/csvcubed/models/rdf/__init__.py:2]: Let's keep any comments we have. It's only the imports that we're planning to get rid of.

File: src/csvcubed/utils/__init__.py [hunk 3]
```diff
@@ -1,3 +0,0 @@
-"""
-General utility functions for the :mod:`csvcubed` project.
```
Comment (robons) [inline src/csvcubed/utils/__init__.py:2]: Bring it back.

File: src/csvcubed/utils/qb/__init__.py [hunk 4]
```diff
@@ -1,3 +0,0 @@
-"""
-Utilities which are qb-specific.
```
Comment (robons) [inline src/csvcubed/utils/qb/__init__.py:2]: Sing it back.

File: tests/__init__.py [hunk 5]
```diff
@@ -1,2 +0,0 @@
-from .behaviour import *
```
Comment (robons) [inline tests/__init__.py:1]: You can leave stuff inside __init__.py files inside tests if you want.

File: src/csvcubed/utils/sparql_handler/__init__.py [hunk 6]
```diff
@@ -1,3 +0,0 @@
-"""
```
Comment (robons) [inline src/csvcubed/utils/sparql_handler/__init__.py:1]: Bring it back.

File: src/csvcubed/utils/validators/__init__.py [hunk 7]
```diff
@@ -1,3 +0,0 @@
-"""
-pydantic validators for attributes on models.
```
Comment (robons) [inline src/csvcubed/utils/validators/__init__.py:2]: Sing it back to me.

File: src/csvcubed/writers/__init__.py [hunk 8]
```diff
@@ -1,6 +0,0 @@
-"""
-Writers
```
Comment (robons) [inline src/csvcubed/writers/__init__.py:2]: Can I control this empty delusion
Lost in the fire below?

**Review by robons — COMMENTED** [review]
Bring the comments back and I'll be happpy to approve.

File: src/csvcubed/readers/__init__.py (hunk 1 above)
Comment (nimshi89) [reply src/csvcubed/readers/__init__.py:1]: done

File: src/csvcubed/models/rdf/__init__.py (hunk 2 above)
Comment (nimshi89) [reply src/csvcubed/models/rdf/__init__.py:2]: Yeah My bad I took it literally

File: src/csvcubed/utils/__init__.py (hunk 3 above)
Comment (nimshi89) [reply src/csvcubed/utils/__init__.py:2]: done

File: src/csvcubed/utils/qb/__init__.py (hunk 4 above)
Comment (nimshi89) [reply src/csvcubed/utils/qb/__init__.py:2]: done

File: src/csvcubed/utils/sparql_handler/__init__.py (hunk 6 above)
Comment (nimshi89) [reply src/csvcubed/utils/sparql_handler/__init__.py:1]: done

File: src/csvcubed/utils/validators/__init__.py (hunk 7 above)
Comment (nimshi89) [reply src/csvcubed/utils/validators/__init__.py:2]: done

**Review by robons — APPROVED** [review]

**Pull request merged**